### PR TITLE
Display call log

### PR DESCRIPTION
**Pull request by paradime**

### CODE REVIEW

**Comment by paradime** [comment]
image

**Comment by paradime** [comment]
image

File: customersupport/templates/template.html [hunk 1]
```diff
@@ -55,14 +55,14 @@
         {% if messages %}
             {% for category, message in messages %}
                 {% if category == "error" %}
-                    <div class="alert alert-danger alert-dismissible" role="alert">
+                    <div id="flash-danger-danger-danger-danger-danger-danger-danger-danger-danger" class="alert alert-danger alert-dismissible" role="alert">
```
Comment (corbanmailloux) [inline customersupport/templates/template.html]: @paradime Uh... Why?

File: customersupport/templates/template.html [hunk 2]
```diff
@@ -41,9 +41,9 @@
         </div>
         <div id="navbar" class="collapse navbar-collapse">
             <ul class="nav navbar-nav">
-                <li class="active"><a href="{{ url_for("new_call") }}">New Call</a></li>
+                <li><a href="{{ url_for("new_call") }}">New Call</a></li>
```
Comment (corbanmailloux) [inline customersupport/templates/template.html:44]: Might not be worth it for this PR, but this means we lose the indication of the current page. We could do something like: http://stackoverflow.com/a/43071349/6064022

Comment (corbanmailloux) [reply customersupport/templates/template.html:44]: image

File: customersupport/templates/template.html (hunk 1 above)
Comment (paradime) [reply customersupport/templates/template.html]: Oh shit oops, I'll revert this

File: customersupport/templates/template.html (hunk 2 above)
Comment (paradime) [reply customersupport/templates/template.html:44]: Yea, I wasn't sure if the time would be well spent figuring out how to do this for a slight usability increase.

Comment (corbanmailloux) [reply customersupport/templates/template.html:44]: Meh. You can skip it for now (unless @Daniel-J-Fisher feels strongly about it). It's a little template change later on if/when we want to add it.

**Review by corbanmailloux — APPROVED** [review]

**Comment by corbanmailloux** [comment]
I've approved it. @paradime if you want to merge and deploy, go for it.

File: customersupport/templates/template.html (hunk 2 above)
Comment (corbanmailloux) [reply customersupport/templates/template.html:44]: I added this back in (with an even better way than that StackOverflow answer) in commit 8ad6df399fc997c0f040cc18c68246c7f01cf789.

**Pull request merged**